testnet/tools/nns-tools/lib/proposals.sh

```diff
@@ -65,8 +65,6 @@ generate_nns_upgrade_proposal_text() {
     local CANDID_ARGS=${4:-}
     local OUTPUT_FILE=${5:-}
 
-    assert_that_a_prebuilt_sns_wasm_is_available "$CANISTER_NAME" "$NEXT_COMMIT"
```

Comment:
We don't still need this assertion at this point?

Reply:
Ah - isn't there an "nns" equivalent?

Reply:
Aha, yes. Nice catch!